tallycash/extension
# Remove goerli feature flag

good bye old friend

## Test
- open wallet 
- go to settings 
- turn on goerli
- check if the network works 

Closes #2489
